colossalai/inference/config.py

```diff
@@ -1,7 +1,73 @@
-"""
-Our config consists of three parts:
-    1. model_config: The configuration for the model, including `model name`, 'model path' and self-defined layer.
-    2. parallel_config: The configuration for parallelize model, including `tp_size`,'pp_size', `world size`, `local rank`, `master port`, `master ip`.
-    3. cache_config: Configuration for initialize and manage kv cache, including `block size`, `block num`
-For the convenience of users, we provide a unified config api for that wrapped all the configs. One can easily construct a colossal_config by setting the needed configs.
-"""
+from typing import Optional, Union
+
+import torch
+import torch.nn as nn
+
+
+class ColossalInferConfig:
```

Comment:
Keep the name sufficient by removing `Colossal`, having `Colossal` as prefix is quite redundant cuz users already import them from `colossalai`. Meanwhile, use `inference` instead of `infer` as `infer` is a verb and `inference` is a noun.

Reply:
Done